webpage/templates/index.html

```diff
@@ -30,11 +32,30 @@ <h1>Grammar and Spelling Checker</h1>
 		<input type="Submit" class="file_submit"></input>
 		
 		<input  type="reset" value="Clear">
-		<br><br>
-		
 
-
-		<label for="outputtext"> Output </label>
+		
+		  <p>
+			Word Count:
```

Comment:
I don't quite understand the purpose of the word count. Would it be better to put it in the corner of the text box?